pkg/attribute/bag_test.go

```diff
@@ -15,6 +15,7 @@
 package attribute
```

Comment:
The attribute package currently has 100% test coverage. This change would reduce this by not testing a considerable number of early exit paths. Please add tests to all these paths so we stay at 100%.